### PR TITLE
fix: Shchedule and details bugs 1 part

### PR DESCRIPTION
**Pull request by ruzniaievdm**
[2U-204](https://youtrack.raccoongang.com/issue/2U-204)
[2U-208](https://youtrack.raccoongang.com/issue/2U-208)
[2U-209](https://youtrack.raccoongang.com/issue/2U-209)
[2U-210](https://youtrack.raccoongang.com/issue/2U-210)

### CODE REVIEW

**Review by PKulkoRaccoonGang — APPROVED** [review]

File: src/generic/internet-connection-alert/index.jsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      setShowAlert(true);
+      onInternetConnectionFailed();
+    } else if (isQueryPending && isOnline) {
+      dispatch(dispatchMethod);
+      onDispatchMethodCall();
+      setShowAlert(false);
+    }
+  }, [isOnline, isQueryPending]);
+
+  useEffect(() => {
```
Comment (sendr) [inline src/generic/internet-connection-alert/index.jsx:41]: why didn't you use the hook above as this one makes the same logic and depends on the same property `isOnline`?

File: src/generic/internet-connection-alert/index.jsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}) => {
+  const intl = useIntl();
+  const dispatch = useDispatch();
+  const [showAlert, setShowAlert] = useState(false);
+  const [isOnline, setIsOnline] = useState(window.navigator.onLine);
+
+  useEffect(() => {
+    const handleOnlineStatus = () => setIsOnline(window.navigator.onLine);
+
+    window.addEventListener('online', handleOnlineStatus);
```
Comment (sendr) [inline src/generic/internet-connection-alert/index.jsx:21]:
```suggestion
    const events = ['online', 'offline'];
    events.forEach(event => {
        window.addEventListener(event, handleOnlineStatus);
    });
```
will it work?

File: src/generic/internet-connection-alert/index.jsx (hunk 1 above)
Comment (ruzniaievdm) [reply src/generic/internet-connection-alert/index.jsx:41]: it looks like this code was used for debugging and it is not needed, removed thanks!

File: src/generic/internet-connection-alert/index.jsx (hunk 2 above)
Comment (ruzniaievdm) [reply src/generic/internet-connection-alert/index.jsx:21]: Yeah, it works well! changed

**Review by sendr — APPROVED** [review]

**Pull request merged**